Source/Scene/Billboard.js

```diff
@@ -762,6 +809,103 @@ define([
         return this._pickId;
     };
 
+    var scratchRay = new Ray();
+    var scratchPosition = new Cartesian3();
+    var scratchCartographic = new Cartographic();
+
+    Billboard._clampPosition = function(object, tile, mode, projection) {
+        var modeChanged = object._mode !== mode;
+        var ellipsoid = projection.ellipsoid;
+        var level = tile.level;
+
+        if (level > object._level || modeChanged) {
+            if (mode === SceneMode.SCENE3D) {
+                Cartesian3.clone(Cartesian3.ZERO, scratchRay.origin);
+                Cartesian3.normalize(object.position, scratchRay.direction);
+            } else {
+                ellipsoid.cartesianToCartographic(object.position, scratchCartographic);
+                scratchCartographic.height = -1000.0; // TODO: get minimum height of entire terrain set
```

Comment:
There's still a `TODO` here.

@abwood might know off the top of his height or check STK's central body file.